app/src/main/java/org/seona/android/sopt/SignUpActivity.kt

```diff
@@ -0,0 +1,66 @@
+package org.seona.android.sopt
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.google.android.material.snackbar.Snackbar
+import org.seona.android.sopt.databinding.ActivitySignUpBinding
+
+class SignUpActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySignUpBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        binding.buttonGoBack.setOnClickListener {
+            val goToLoginActivity = Intent(applicationContext, LoginActivity::class.java)
+            startActivity(goToLoginActivity)
+        }
+
+        binding.buttonSignup.setOnClickListener {
+            if (binding.etSingupId.text.length < 6 || binding.etSingupId.text.length > 10) {
```

Comment:
```suggestion
            if (binding.etSingupId.text.length !in 6..10) {
```
이렇게 코틀린 방식으로 쓰면 코드 길이를 줄일 수 있습니다!